wbooks/App.tsx

```diff
@@ -1,3 +1,9 @@
+if (__DEV__) {
+  import('./src/config/reactotronConfig').then(() => {
+    // eslint-disable-next-line no-console
+    console.log('Reactotron Configured');
+  });
+}
```

Comment:
Not necessary the `__DEV__` here because you already put inside `reactotronConfig`:
```
import React from 'react';

import './src/config/reactotronConfig';
import App from './src/app';
```